linfa-ensemble/src/voting_classifier/algorithm.rs

```diff
@@ -0,0 +1,109 @@
+use linfa_predictor::{LinfaError, Predictor};
+use ndarray::{Array1, ArrayBase, Data, Ix2};
+use std::collections::HashMap;
+
+#[derive(Clone)]
+pub enum Voting {
+    Hard,
+    // Soft,
```

Comment:
nit: If it's not implemented yet that's fine, we should document it in the roadmap instead of here where it's easy to forget about.